File: c dsa codes/linkedlist.c
#include<stdio.h>
#include<stdlib.h>
struct node{
    int data;
    struct node* Next ;
};

// Linked list traversal
void linkedlistTraversal(struct node* ptr){
    while (ptr != NULL)
    {
        printf("Element: %d\n", ptr->data);
        ptr = ptr->Next;
    }
}

int main(){
// Linked list making
    struct node* head;
    struct node* second;
    struct node* third;

    head=(struct node*)malloc(sizeof(struct node));
    second=(struct node*)malloc(sizeof(struct node));
    third=(struct node*)malloc(sizeof(struct node));

    head->data=32;
    head->Next=second;

    second->data=22;
    second->Next=third;

    third->data=11;
    third->Next=NULL;

    linkedlistTraversal(head);

    return 0;
}

File: java dsa codes/linkedlist.java
public class linkedlist {

    static class Node{
        int data;
        Node next;
    }

    public static void linkedlisttraversl(Node head){
        while(head!=null){
            System.out.println(head.data);
            head=head.next;
        }
    }
    public static void main(String [] args){
        Node head=new Node();
        head.data=11;
        head.next=null;

        Node second=new Node();
        second.data=22;
        second.next=null;

        Node third=new Node();
        third.data=45;
        third.next=null;

        head.next=second;
        second.next=third;

        linkedlisttraversl(head);

    } 
    
}
